ui/src/shared/components/TableGraph.js

```diff
@@ -89,6 +89,7 @@ class TableGraph extends Component {
       this.setState({
         timeColumnWidth: calculateTimeColumnWidth(timeFormat),
       })
+      this.multiGridRef.forceUpdateGrids()
```

Comment:
Is this forceUpdateGrids() necessary since the component is already updating?

Reply:
It actually changes the size of all the columns except the time column. If you don't have enough columns to fill the table you'll see a gap on the right side that changes in size depending on the time format. Without `.forceUpdateGrids()` the gap appears and stays until you refresh the page

Reply:
Apparently `react-virtualized` has some issue in which it doesn't treat cells in lower right grid the same as the fixed rows/columns